src/core/server/saved_objects/service/lib/search_dsl/query_params.ts

```diff
@@ -127,6 +127,26 @@ function getClauseForType(
     },
   };
 }
+/**
+ *  Gets the clause that will filter for the workspace.
+ */
+function getClauseForWorkspace(workspace: string) {
+  if (workspace === '*') {
```

Comment:
should we check if workspace is undefined or ""?

Reply:
Yes, we should do that. Done.